crypto/src/bls12377/mod.rs

```diff
@@ -42,7 +42,7 @@ pub struct BLS12377PublicKey {
     pub bytes: OnceCell<[u8; CELO_BLS_PUBLIC_KEY_LENGTH]>,
 }
 
-pub type BLS12377PublicKeyBytes = PublicKeyBytes<BLS12377PublicKey, { BLS12377PublicKey::LENGTH }>;
+pub type BLS12377PublicKeyBytes = PublicKeyBytes<BLS12377PublicKey, { CELO_BLS_PUBLIC_KEY_LENGTH }>;
```

Comment:
We should avoid including project names in fixed variable names. Let's keep it as it was before.

Reply:
```suggestion
pub type BLS12377PublicKeyBytes = PublicKeyBytes<BLS12377PublicKey, { BLS12377PublicKey::LENGTH }>;
```